cloudferrylib/os/identity/keystone.py

```diff
@@ -456,4 +456,10 @@ def get_dst_user_from_src_user_id(src_keystone, dst_keystone, src_user_id,
         return dst_user
     except keystoneclient.exceptions.NotFound:
         LOG.warning("User '%s' not found on DST!", src_user_name)
-        return
+        if fallback_to_admin:
+            LOG.warning("Replacing user '%s' with DST admin", src_user_name)
+            dst_user = dst_keystone.keystone_client.users.find(
+                name=cfglib.CONF.dst.user)
+            return dst_user
```

Comment:
There's no need to search for user with keystone, you can simply return `cfglib.CONF.dst.user` here (everything would fall apart way earlier without admin user available on destination).